docs/prow/dependency-update.md

```diff
@@ -0,0 +1,68 @@
+# Dependency updates
+
+To help keep dependency up to date you can use dependabot together with automated testing and merging of pull requests.
```

Comment:
```suggestion
To help keep dependency up to date you can use Dependabot together with automated testing and merging of pull requests.
```
This seems to be a proper name that's capitalized in the original documentation.